### ISSUE TITLE
Document download url adjustment

### ISSUE THREAD

**Issue by damm89** [GemeenteUtrecht/zaakafhandelcomponent#159]
For further integration of the "document wijzigen component" (_doc_) we need some extra buttons for the documents INSTEAD of the document download button.

This is the workflow for what needs to be facilitated with regards to documents:
- A user should be able to open a document for READING only: <button>Lezen</button>
- A user should be able to WRITE a document: <button>Wijzigen</button>
> - A user document should be able to finish the document write: <button>Klaar met wijzigen</button> (only visible if someone has clicked wijzigen)

I.e., instead of a single download button we need two buttons: Open & Wijzigen and when a user clicks Wijzigen, the button "Klaar met wijzigen" must show.

When a user clicks a document button the ZAC does a post request to the _doc_. If the request is succesful the _doc_ returns a JSON. There are three links they can click:

READ a document (so under Lezen button):
Request: `POST /api/dowc/<str:bronorganisatie>/<str:identificatie>/read`
Success response:
```
{
    'purpose':'read',
    'magicUrl':'http://some-magic-url.com/with-magic/',
    'deleteUrl':'',
}
```

WRITE a document (so under Wijzigen button):
Request: `POST /api/dowc/<str:bronorganisatie>/<str:identificatie>/write`
Success response:
```
{
    'purpose':'read',
    'magicUrl':'http://some-magic-url.com/with-magic/',
    'deleteUrl':'http://some-delete-url.com/that-patches-and-destroys-the-document-in-the-doc/',
}
```

DELETE a document (so under Klaar met wijzigen button):
Request: `DELETE /api/dowc/<uuid>/`
Success response: HTTP_204

This is for a first iteration. For later iterations we need to somehow be able to feed the user the right set of buttons the moment they load the page. For example, what if they were already editing the document but never clicked the "Klaar met wijzigen" button... We would need to implement in collaboration with the rest of the team some sort of default behaviour.

A proposal: they would immediately see the "Klaar met wijzigen" button or "Doorgaan met wijzigen" button instead of all other buttons. This is for (a) future sprint(s).

**Comment by damm89** [comment]
Check: http://localhost:8080/api/docs/#operation/dowc_create in the feature/document-wijzigen branch for api docs.

**Comment by sergei-maertens** [comment]
Fixed via #157
